lib/openstax/rescue_from/deploy_utils.rb

```diff
@@ -0,0 +1,18 @@
+module OpenStax
+  module RescueFrom
+    class DeployUtils
```

Comment:
Done by just making the dang name of the thing configurable without need for matching code stuff